app/controllers/assignments_controller.rb

```diff
@@ -759,6 +757,7 @@ def assignment_params
         :group_name_displayed,
         :invalid_override,
         :section_groups_only,
+        :is_required,
```

Comment:
This attribute name isn't very good. I would suggest "only_required_files" instead.